pkg/options/options.go

```diff
@@ -6,97 +6,70 @@
 package options
 
 import (
-	"encoding/json"
-	"reflect"
 	"sort"
 	"strings"
 
+	"github.com/DataDog/pupernetes/pkg/util/sets"
 	"github.com/fatih/structs"
 	"github.com/golang/glog"
 )
 
+var commonOptions = sets.NewString("all", "none")
+
 type common struct {
 	All  bool `json:"all,omitempty"`
 	None bool `json:"none,omitempty"`
 }
 
-func containsString(slice []string, elt string) bool {
-	for _, item := range slice {
-		if elt == item {
-			return true
-		}
-	}
-	return false
+// GetCleanOptionsString returns a string representation of available "--clean" options.
+func GetCleanOptionsString() string {
+	return getOptionsString(Clean{})
 }
 
-func setAllOptionsTo(d interface{}, set bool) {
-	for _, name := range structs.Names(d) {
-		if name == "common" {
-			continue
-		}
-		reflect.ValueOf(d).Elem().FieldByName(name).SetBool(set)
-	}
+// GetDrainOptionsString returns a string representation of available "--drain" options.
+func GetDrainOptionsString() string {
+	return getOptionsString(Drain{})
 }
 
-// GetOptionNames returns the options from the given interface
-// The interface must be a Clean or Drain one
-func GetOptionNames(opt interface{}) string {
-	var names []string
-	for _, elt := range structs.Names(opt) {
-		elt = strings.ToLower(elt)
-		if elt == "common" {
-			continue
-		}
-		names = append(names, elt)
-	}
+func getOptionsString(opt interface{}) string {
```

Comment:
Why not making this public directly?

Reply:
I just follow the pattern that everything should be private until it can't be. Since I added `GetDrainOptionsString` and `GetCleanOptionsString` there was no packages that call `getOptionsString` so I made it private.